data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2378 @@
+		"Remass Injector" 4
+		"XY-360 Radial Thrusters"
+		"Hyperdrive"
+
+	turret "Speckle Turret"
+	turret "Speckle Turret"
+	turret
+	turret
+	turret
+	turret
+	turret "Green Optical Laser"
+	turret "Green Optical Laser"
+
+
+ship "Refraktos"
+	plural "Refraktos"
+	sprite "ship/avgi refraktos"
+	thumbnail "thumbnail/avgi refraktos"
+	attributes
+		category "Medium Warship"
+		licenses
+			"Twilight Guard"
+		"cost" 4560000
+		"shields" 3600
+		"hull" 8400
+		"required crew" 12
+		"bunks" 24
+		"mass" 540
+		"drag" 6
+		"heat dissipation" 1.8
+		"fuel capacity" 1800
+		"cargo space" 60
+		"outfit space" 516
+		"weapon capacity" 150
+		"engine capacity" 144
+		"anchor point" 1
+		weapon
+			"blast radius" 420
+			"shield damage" 880
+			"hull damage" 440
+			"hit force" 1320
+	outfits
+		"Blue Optical Laser" 4
+		"Green Optical Laser" 3
+
+		"Photoelectric Lamina"
+		"Ultracapacitor Bank" 3
+		"Diffuse Deflector"
+		"Optical Lasing Generator" 5
+		"Multispectral Scanner"
+		"Electrostatic Ramscoop"
+		"Thermal Sink Partition"
+		"Electromotive Rifle" 24
+		
+		`Z-660 "Zap" Fusion Torch`
+		"Inductive Extractor" 5
+		"Remass Injector" 1
+		"XY-360 Radial Thrusters"
+		"Hyperdrive"
+
+	gun -24 -33
+		angle -90
+	gun 24 -33
+		angle 90
+	gun -24 -9
+		angle -90
+	gun 24 -9
+		angle 90
+	turret -18 -81 "Blue Optical Laser"
+	turret 18 -81 "Blue Optical Laser"
+	turret -18 -72 "Blue Optical Laser"
+	turret 18 -72 "Blue Optical Laser"
+	turret -21 -60 "Green Optical Laser"
+	turret 21 -60 "Green Optical Laser"
+	turret 0 54 "Green Optical Laser"
+
+	engine 0 150
+		gimbal 6
+	"reverse engine" -9 162
+		angle -30
+	"reverse engine" 9 162
+		angle 30
+	"steering engine" -12 -108
+		angle 90
+		right
+	"steering engine" 12 -108
+		angle -90
+		left
+	"steering engine" -9 162
+		angle 90
+		left
+	"steering engine" 9 162
+		angle -90
+		right
+
+	leak "leak" 60 60
+	leak "flame" 60 90
+	leak "big leak" 30 60
+	explode "tiny explosion" 12
+	explode "small explosion" 18
+	explode "medium explosion" 18
+	explode "large explosion" 6
+	"final explode" "final explosion medium"
+
+	description "The Refraktos-class Laserstar is a relatively recent, post-Diaspora destroyer design with a single focus: directed energy weapons. While its armor and shielding systems are light for its size, its expansive radiators can support the thermal loads of a staggering array of laser weapons, both offensive and defensive, allowing it to destroy both missiles and small ships from a vast distance."
+	description "A long truss running the length of the ship is built to fit an impressively large mirror-cell fusion torch, giving the Refraktos the mobility needed to evade opponents until they are reduced to hot vapor."
+	description "Due to the anemic damage output of Avgi laser weapons compared to their kinetic weapons and missiles, the Refraktos is most often seen in a support role, providing antimissile coverage to entire fleets."
+
+
+# Heavy Warships
+
+ship "Atenuatia"
```

Comment:
I assume that the Atenuatia won't be player-usable for a long while yet, but I'd still be a little concerned about its number of carriables in that eventuality. 24 is a LOT, outdoing the next-largest carriers by a substantial step (8 more than the TF109, and 10 more than that one PR makes the Navy Carrier). Perhaps I'm always biased towards carriers-as-freighters, but the sheer amount of freight that could be carried with properly-outfitted Ateunatia and carriables gives me pause. Maybe tone down the drone count somewhat, from- say- 18/6 to 12/6? On the other hand, there's something to be said for warships-turned-plowshares as a natural expression of player cursedness/creativity....

Reply:
> the sheer amount of freight that could be carried with properly-outfitted Ateunatia and carriables gives me pause.

Having finally remembered ESL2 exists and I can actually run a PR, I can give you numbers on this now. Completely stock, an Atenuatia with six Akoustiki and eighteen Sonikis can haul 1,272 tons- a ways below the capacity of a Republic Navy Carrier's four fighters and six drones. With the carriables refit for full cargo hauling, this jumps to 2,172 tons- somewhat below the seven fighters and nine drones of a Tek Far 109, or somewhat above the capacity of the Republic Navy Carrier if that one PR gets in. With the Atenuatia refit for cargo hauling (using three External Launch Tubes for piracy deterrence, a Spark instead of a Zap, and nearly all other outfits removed) it's all the way to 2,640 tons- a mid-sized freighter's worth above the TF109.

It's probably pretty hard to collect eighteen Faes'mari. If we keep the drones as Sonikis, the cargo capacity climbs slightly more to 2,790 tons. But if we run Faes'mari, the cargo capacity is a horrifying 4,071 tons (once we've removed a cargo expansion to replace it with a fourth External Launch Tube for piracy deterrence). Obviously this capacity is probably not quite what one would achieve in practice, assuming they wish to use frivolities such as shield generators, but it is still extremely high. It will be a long time before the Atenuatia is for sale, or even capturable (I imagine); but being nearly four times the capacity of a Deep River or three times the capacity of a Karek A'awoj is quite the feat. 

My initial idea of making the count 6 fighters and 12 drones drops it to 3,099 tons, which is a bit over twice that of a suitably-outfitted Navy Carrier. Making it 5 fighters and 15 drones (in case you want some sort of one fighter leading three drones thing) makes it 3,470 tons. 

(Just for fun- outfitting them all for _passengers_ instead gives 944 bunks, most of which are of course unusable in boarding. Just in case you wanted to haul passengers in missile storage, ore bays, and lifepod emergency storage.)

Reply:
One more just for fun aside, when similarly outfitted:
Difraktos- 684 tons
Diaspersi- 1632 tons (again comparable to the current Carrier maximum)
Undsyni- 1608 tons (ditto)
Polari- 601 tons (but the Polari doesn't have the weapons space or gun ports to eliminate its piracy threat)
Entasi- 1697 tons 
Modified Tachytia- 4061 tons (not enough weapons space or gun ports to eliminate piracy threat)

...Wait. A Smew can fit an External Launch Rail without needing to remove a cargo expansion. Okay, it's 4181 tons on the Modified Tachytia and 4131 on the Atenuatia.